### PR TITLE
Johann firebase cloud messaging

### PR DESCRIPTION
**Pull request by JohannBillar**
So far -> notifying all users with a token when a new DoWhop description has been created.

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+    const tokens = [];
+    snapshot.forEach(user => {
+      const token = user.child('token').val();
+      if (token) tokens.push(token);
+    });
+    return tokens;
+  });
 
-// TODO(DEVELOPER): Write the blurOffensiveImages Function here.
+  const getDoer = admin.auth().getUser('VYw0lPDFD3btHJadneuSFGjy8wk1');
```
Comment (omarchaeopteryx) [inline functions/index.js]: Should this be hard-coded here? Or is this just for debugging purposes?

Comment (TinkerDo) [inline functions/index.js]: We have a blur offensive images function!?

Comment (JohannBillar) [inline functions/index.js]: Here's a list of Cloud functions in beta. https://github.com/firebase/functions-samples

Comment (JohannBillar) [inline functions/index.js]: @omarchaeopteryx Hard coded to test the getUser function.

**Pull request merged**